snakemake/utils.py

```diff
@@ -618,6 +618,9 @@ def __init__(self, dataframe, filename_params=None, param_sep="~"):
             self.pattern = "/".join([r"{}"] * len(self.dataframe.columns))
             self.ordered_columns = self.dataframe.columns
         else:
+            if isinstance(filename_params, bool) and filename_params:
```

Comment:
I think it would be more readable if we require a filename_params to be a string `"*"`, instead of true. Or, alternatively, we introduce another keyword argument for this behavior, like `only_filename=True` or so.

Reply:
Adding a new keyword argument would be very specific but also increase complexity more than making `filename_params` more flexible (I think?).
So I'll make us of `"*"`.